rootston/config.c

```diff
@@ -182,6 +221,15 @@ struct roots_config *parse_args(int argc, char *argv[]) {
 	struct roots_config *config = calloc(1, sizeof(struct roots_config));
 	wl_list_init(&config->outputs);
 	wl_list_init(&config->devices);
+	wl_list_init(&config->bindings);
+
+	// TEMPORARY, probably
```

Comment:
Temporary?

Reply:
Oh, I misunderstood this. Take this out, please.

Reply:
Let's only add a sane default quit binding if there's no config file (make it logo+q though).